src/main/java/uk/gov/hmcts/reform/notificationservice/data/NotificationRepository.java

```diff
@@ -52,13 +52,15 @@ public NotificationRepository(
     }
 
     public List<Notification> find(String zipFileName, String service) {
-        return jdbcTemplate.query(
-            "SELECT * FROM notifications WHERE zip_file_name = :zipFileName AND service = :service",
+        var temp = jdbcTemplate.query(
```

Comment:
Do not need local variable here

Reply:
I agree. I was debugging and left it there mistakenly. Consider it done.

Reply:
All changes made as sugested.